### PR TITLE
Splunk improvements

### PR DESCRIPTION
**Pull request by tusbar**

### CODE REVIEW

File: playbooks/roles/splunkforwarder/tasks/main.yml [hunk 1]
```diff
@@ -87,7 +87,7 @@
 # Drop template files.
 - name: splunkforwarder | drop input configuration
   template:
-    src=opt/splunkforwarder/etc/system/local/inputs.conf.j2
```
Comment (e0d) [inline playbooks/roles/splunkforwarder/tasks/main.yml]: We've standardized on using the full path to the location the file will ultimately be placed in the template directory.  We haven't universally fixed all older templates, but we don't want to remove it where we have it in place.

**Comment by feanil** [comment]
@tusbar  the reason for the seperation of the splunk package location and splunk package URL is so that we can easily seperate the location of the deb and the name/version of it.  This ensures that geturl will only run once per splunkforwarder version.

**Comment by tusbar** [comment]
@feanil I don’t mind the separation, there still are two variables in this pull request, but the URL doesn’t have to end with the deb’s name (and that’s not the case when you’re downloading from Splunk’s website).

File: playbooks/roles/splunkforwarder/tasks/main.yml (hunk 1 above)
Comment (tusbar) [inline playbooks/roles/splunkforwarder/tasks/main.yml]: Alright. It just seemed to contradict the fact that there is a variable to specify splunkforwarder’s installation path.

**Comment by hkim823** [comment]
@e0d and @feanil cleaning up older PRs that fell threw the cracks, any thoughts on this?

**Comment by e0d** [comment]
I'm fine with it, but it will definitely need a rebase.

**Comment by feanil** [comment]
Agreed.

**Comment by tusbar** [comment]
@e0d rebased.

**Pull request merged**